jobs/germany-crowd-forecast-data-update.sh

```diff
@@ -0,0 +1,16 @@
+#!bin/bash
+
+# check repository is up to date
+docker exec -w /home/rstudio/covid-german-forecasts forecast git pull -Xours
+
+# update dependencies
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript -e 'devtools::install_github("epiforecasts/covid-german-forecasts")'
+
+# Redeploy forecast app (to update data)
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript human-forecast/redeploy-forecast-app.R
+
+# rerun evaluation / leaderboard
+docker exec -w /home/rstudio/covid-german-forecasts forecast Rscript evaluation/forecast-evaluation.R
```

Comment:
again repeat naming.

Reply:
I like it when I have the file open and I know what it does, even if I don't know the folder it sits in :octopus: 
I'm already getting confused by the fact that I have 2 app.R scripts....

Reply:
but happy to oblige ;)